apps/dashboard/src/pages/group.tsx

```diff
@@ -52,11 +52,15 @@ export default function GroupPage(): JSX.Element {
     const addMembersModal = useDisclosure()
     const toast = useToast()
     const { groupId, groupType } = useParams()
+    const {adminId, adminType } = useParams()
     const [_group, setGroup] = useState<Group | null>()
     // const { hasCopied, setValue: setApiKey, onCopy } = useClipboard("")
     const { hasCopied: hasCopiedGroupId, onCopy: onCopyGroupId } = useClipboard(
         groupId || ""
     )
+    const { hasCopied: hasCopiedAdminId, onCopy: onCopyAdminId } = useClipboard(
+        adminId || ""
+    )
```

Comment:
```suggestion
       const { hasCopied: hasCopiedAdminId, onCopy: onCopyAdminId } = useClipboard(
        (_group && _group.admin) || ""
    )
```

I think we can remove the following line: `const {adminId, adminType } = useParams()` and update this `useClipboard` hook and add it after `const isGroupAdmin`.

Reply:
done @vplasencia